script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_5.txt

```diff
@@ -32,7 +32,7 @@ Almost there.
 Arriving at the corner and with my feet facing north, ---on the left, there lies...
 <Page11>
 The Tohno mansion, towering high, like a great wall of chalk.
-Ah, I let out a sigh that could be described as both admiration and resignation.
+Ah, I let out a sigh that could be either admiration and resignation.
```

Comment:
Should be "either admiration **or** resignation", "either .. and" doesn't really make sense
Also - perhaps "a sigh that could be **of** either "
(As in, final corrected line perhaps "Ah, I let out a sigh that could be of either admiration or resignation.")